### DISCUSSION TITLE
Reverting LTMEAT

### DISCUSSION THREAD

**Discussion by Cooldunkin8** [3kh0/ext-remover#683] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
Instead of restarting your whole Chromebook, you could just bookmark chrome://restart and press it whenever you want to revert it

**Comment by jessiejs** [comment]
Or just go to the extension and toggle file URL access.

**Comment by KandyKaneMarvel** [comment]
That's LTBeef

**Comment by jessiejs** [comment]
Y-you do realize that just because you can also do that for ltBeef, doesn't make it not work for LTMEAT. It's just a way to restart extensions, correct me if I'm wrong but I'm pretty sure that the hang doesn't affect the restart.

**Reply by Aka-but-nice** [reply]
MY GUY IS WOKE!

**Reply by Aka-but-nice** [reply]
I was saved 2 minutes of shaming this delinquent

**Reply by jessiejs** [reply]
not the wokism!!!!
